src/libstd/keyword_docs.rs

```diff
@@ -558,6 +558,32 @@ mod impl_keyword { }
 /// [Reference]: ../reference/statements.html#let-statements
 mod let_keyword { }
 
+#[doc(keyword = "while")]
+//
+/// while loop
+///
+/// `while` is used to define while loops in Rust. It runs the code inside it
+/// until the condition in the while loop becomes False or there the code uses
+/// `break`.
+///
+/// ```rust
+/// while true {
+///     println!("hello world forever!");
+///     # break;
```

Comment:
Maybe remove this line

Reply:
The `#` causes a compiler failure, if you want to leave it here you must use `// break;`